### PR TITLE
Layout/dashboard clock in out

### PR DESCRIPTION
**Pull request by herulhidayat**
Did you write good & comprehensive specs? ⭐️

What's this PR do ? ⭐️
Add layout for dashboard clock in clock out
What are the relevant tickets ? ⭐️
https://www.notion.so/prawito/Create-Clock-in-clock-out-page-for-front-end-3641cd26d2aa4210b7182aad57e6d20d
Where should the reviewer start ?

How should this be manually tested ?

What is the Dockup URL ?

Any background context you want to provide ?

Any deployment note (migrations, new envs, rake tasks, cron) ?

Screenshots (if appropriate)
image

Definition of Done:

[ x ] Specs / Tests are adequate ? ⭐️

[ x ] Changes conform with code quality tools (Rubocop, Reek, Brakeman) ? ⭐️

[ x ] Does this PR require Database changes ?

[ x ] Does this PR require monitoring after deployment (Scout, logs, Sentry, etc) ?

[ x ] Does this PR require a regression test by QA ? All fixes require a regression test.

[ v ] Does this add new dependencies ? (network setup, new 3rd party services)

### CODE REVIEW

File: src/Components/AttendanceClock.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Jumbotron, Card, CardBody, Container, Row, Col, Button} from 'reactstrap';
+import Clock from 'react-live-clock';
+import { AttendanceLog } from './AttendanceLog';
+
+const AttendanceClock = (props) => {
+    const { user, users } = props; 
+    console.log(users);
+    return (
+    <>
```
Comment (rahmatullah5) [inline src/Components/AttendanceClock.js:15]: what is this for ?

File: src/Components/AttendanceLog.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+// import './AttendanceLog.css';
```
Comment (rahmatullah5) [inline src/Components/AttendanceLog.js:2]: please remove unused code.

**Review by rahmatullah5 — CHANGES_REQUESTED** [review]

File: src/Components/AttendanceClock.js (hunk 1 above)
Comment (herulhidayat) [reply src/Components/AttendanceClock.js:15]: "<>" is a short syntax from "<React.Fragment>"

File: src/Components/AttendanceClock.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { userActions } from '../Actions';
+import '../Styles/Dashboard.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Jumbotron, Card, CardBody, Container, Row, Col, Button} from 'reactstrap';
+import Clock from 'react-live-clock';
+import { AttendanceLog } from './AttendanceLog';
+
+const AttendanceClock = (props) => {
+    const { user, users } = props; 
+    console.log(users);
```
Comment (rahmatullah5) [inline src/Components/AttendanceClock.js:13]: Please remove console log from production.

File: src/Components/TableComponent.js [hunk 4]
```diff
@@ -81,7 +81,7 @@ const defaultSorted = [
 
 const TableComponent = (props) => {
   return (
-    <Container>
+    <Container className="container-set">
       {/* <BootstrapTable
```
Comment (rahmatullah5) [inline src/Components/TableComponent.js:85]: Please remove unused code

Comment (rahmatullah5) [inline src/Components/TableComponent.js:85]: remove unused code

**Review by rahmatullah5 — APPROVED** [review]

**Review by chaterinamf — APPROVED** [review]

**Review by fknraini — APPROVED** [review]

**Pull request merged**